src/mlpack/tests/rl_components_test.cpp

```diff
@@ -57,4 +58,34 @@ BOOST_AUTO_TEST_CASE(SimpleCartPoleTest)
   BOOST_REQUIRE_EQUAL(2, CartPole::Action::size);
 }
 
+/**
+ * Construct a random replay instance and check if it works as
+ * it should be.
+ */
+BOOST_AUTO_TEST_CASE(RandomReplayTest)
+{
+  RandomReplay<MountainCar> replay(1, 1);
```

Comment:
I was thinking it's probably a good idea to test RandomReplay with capacity > 1. We could write a new test or integrate that into the existing.